dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -900,6 +938,44 @@
                     "REOPEN"
                   ]
                 },
+                "specialIssues": {
```

Comment:
This is in singular case in the swagger: `specialIssue`

Reply:
In the payload, EVSS expects it to be "specialIssues"

<img width="1359" alt="screen shot 2018-11-15 at 4 41 27 pm" src="https://user-images.githubusercontent.com/11605414/48583531-5a097180-e8f5-11e8-913e-1d39a02ce2bc.png">

Reply:
That link for those errors points towards the original version evss endpoint (ie v1) which is at this url:
https://csraciapp6.evss.srarad.com/wss-form526-services-web/swagger-ui/index.html?url=https://csraciapp6.evss.srarad.com/wss-form526-services-web/rest/swagger.yaml#!/form526/submit

that one has it setup like you mentioned. But the new version of their endpoint (ie v2) has it as I described in my comments, located here:
https://csraciapp6.evss.srarad.com/wss-form526-services-web-v2/swagger-ui/index.html?url=https://csraciapp6.evss.srarad.com/wss-form526-services-web-v2/rest/swagger.yaml#!/form526/submit

Are they planning on reverting their changes?

Reply:
@SMLuthi @cvalarida : The changes are now completed to sync up with the v2 for all claims. To be singular `specialIssue` Please review